DuckDuckGo/NavigationBar/View/AddressBarTextField.swift

```diff
@@ -961,6 +963,7 @@ extension AddressBarTextField: NSTextFieldDelegate {
 
     func controlTextDidChange(_ obj: Notification) {
         handleTextDidChange()
+        onboardingDelegate?.trackAddressBarTypedIn()
```

Comment:
perhaps this should be inverted?

AddressBarTextField shouldn’t know about / be coupled with onboarding. We could have an AddressBarTextFieldDelegate protocol with a method `didChangeControlText`. In that way we send the pixel also from a VC or VM (if it existed :D) rather than from a view.

Let me know what you think.